lib/helpers.js

```diff
@@ -38,16 +38,16 @@ function getTheme(config, selected) {
     };
 }
 
-function getConfig() {
-    return yaml.safeLoad(fs.readFileSync(CONFIG_FILE_PATH), 'utf8');
+function loadConfig(file) {
+    return yaml.safeLoad(fs.readFileSync(path.join(CONFIG_DIR_PATH, file)), 'utf8');
 }
 
-function getConfigPath() {
-    return CONFIG_FILE_PATH;
+function getConfigFilesPath() {
+    return path.join(CONFIG_DIR_PATH, '_files.yml');
 }
 
 function generateDataJson() {
-    const config = getConfig();
+    const config = require('../config'); // eslint-disable-line global-require
```

Comment:
Circular dependancies. You can't load configuration at the top of the file, because `config/index.js` requires this file. The require needs to be done on demand, once helpers is fully loaded. This could be fixed by having `generateDataJson` accept configuration as a param, but I figured this sufficed for now.